docs/FlattenRecord-transform.md

```diff
@@ -1,13 +1,16 @@
-# Flatten Transform
+# Flatten Record Transform
 
 ## Description
-Flatten is a transform plugin that flattens nested data structures.
+FlattenRecord is a transform plugin that flattens nested data structures.
 
 ## Use Case
 
 The transform is used  to convert a nested data structure  into a single flattened record where each key in the record 
 is a ```_``` name path to the node in the nested structure. 
 
+Note: the plugin supports only flattening of records. In order to flatten an array of records first use Wrangler 
+Flatten directive to flatten the array and then use FlattenRecord transform.
```

Comment:
could we add link to the directive: https://github.com/data-integrations/wrangler/blob/develop/wrangler-docs/directives/flatten.md